pipelines/scipipe/ap_verify.groovy

```diff
@@ -270,10 +270,20 @@ def void verifyDataset(Map p) {
       // push results to squash
       if (p.squashPush) {
         switch (conf.gen) {
+          case 3:
+            // Partially hard-coded in ap_verify
+            def gen3Dir = util.joinPath(runDir, ds.name, "repo")
+            util.runGen3ToJob(
+              gen3Dir: gen3Dir,
+              collectionName: "ap_verify-output",
+              namespace: "",
```

Comment:
This could change later?

Reply:
`namespace` is unlikely to change for `ap_verify` short of a massive, breaking `verify`-level reorganization of all pipeline metrics. However, it's needed by `validate_drp`.

`collectionName` could in principle change; currently it's hard-coded into `ap_verify`, and certainly wouldn't be expected to change _dynamically_.